frame/staking/src/tests.rs

```diff
@@ -3256,12 +3255,18 @@ fn non_slashable_offence_doesnt_disable_validator() {
 			&[Perbill::zero()],
 		);
 
+		// it does NOT affect the nominator.
+		assert_eq!(Staking::nominators(101).unwrap().targets, vec![11, 21]);
+
 		// offence that slashes 25% of the bond
 		on_offence_now(
 			&[OffenceDetails { offender: (21, exposure_21.clone()), reporters: vec![] }],
 			&[Perbill::from_percent(25)],
 		);
 
+		// it DOES affect the nominator.
+		assert_eq!(Staking::nominators(101).unwrap().targets, vec![11]);
```

Comment:
So we still chill nominations? :confused:

Reply:
assertions are outdated. Updated now.